core/src/main/java/org/stellar/anchor/util/SepHelper.java

```diff
@@ -47,6 +48,19 @@ public static String memoTypeString(MemoType memoType) {
     return result;
   }
 
+  public static MemoType memoType(Memo memo) throws SepException {
+    if (memo instanceof MemoId) {
+      return MEMO_ID;
+    } else if (memo instanceof MemoHash) {
+      return MEMO_HASH;
+    } else if (memo instanceof MemoText) {
+      return MEMO_TEXT;
+    } else if (memo instanceof MemoNone) {
+      return MEMO_NONE;
+    }
+    throw new SepException("Unsupported memo type: " + memo.getClass());
+  }
+
```

Comment:
©️  We already have a very similar function in MemoHelper. Could you reuse that? And feel free to change it to your needs.

https://github.com/stellar/java-stellar-anchor-sdk/blob/29c7ef66e94c0b9503ca68e15b07da064a76ee2d/core/src/main/java/org/stellar/anchor/util/MemoHelper.java#L35-L47

Reply:
Cool. I had merged the two functions.